pygsheets/worksheet.py

```diff
@@ -1710,6 +1710,98 @@ def set_data_validation(self, start=None, end=None, condition_type=None, conditi
             request['setDataValidation']['rule'] = rule
         self.client.sheet.batch_update(self.spreadsheet.id, request)
 
+    @batchable
+    def set_basic_filter(
+        self,
+        start_pos=None,
```

Comment:
start_pos -> start, for consistency
end_pos -> end

Reply:
I fixed with the commit 374483e